bind/flag.go

```diff
@@ -25,6 +27,13 @@ import (
 	"github.com/spf13/pflag"
 )
 
+type Format int
```

Comment:
Let's name it `DescribeFormat`. This is `bind` package and `bind.Format` might be misleading.